docs/components/index.md

```diff
@@ -1,26 +1,156 @@
 <script setup>
-const components = import.meta.glob('./*/data.json', { eager: true })
-const baseUrl = import.meta.env.BASE_URL
+import { ref, computed } from 'vue';
 
-const componentData = Object.keys(components).map(path => {
- return ({
-  ...components[path].default, // component data from JSON
+// Import all JSON data
+const components = import.meta.glob('./*/data.json', { eager: true });
+const baseUrl = import.meta.env.BASE_URL;
+
+// Map JSON data to include href and correct paths
+const componentData = Object.keys(components).map(path => ({
+  ...components[path].default,
   href: `${baseUrl}components/${path.replace('/data.json', '')}`
-})});
+}));
+
+// Reactive variables for query and selected frameworks
+const query = ref('');
+const selectedFrameworks = ref([]);
+
+// Extract unique framework names
+const frameworkNames = computed(() => {
+  const names = new Set();
+  componentData.forEach(component => {
+    component.frameworks.forEach(framework => {
+      names.add(framework.name);
+    });
+  });
+  return Array.from(names);
+});
+
+// Computed property to filter components based on query and selected frameworks
+const filteredComponents = computed(() => {
```

Comment:
I'm not sure this works properly, because when I run the app from this branch and selected both React and Elements framework in the filters, I still saw Radio and Checkbox component, and those are not supported in Elements 🤔 When I only select Elements, then these component cards are filtered out.

Reply:
Dunno, no garantee, I didnt check towards the actual data in the json :D Just saw that they were filtering something and presumed it was correct :D